AAEmu.Game/Models/Game/Char/Character.cs

```diff
@@ -1484,6 +1504,39 @@ public static Character Load(uint characterId)
                 return Load(connection, characterId);
         }
 
+        public uint MaxBreath
+        {
+            get
+            {
+                if (Race == Race.Elf)
+                    return 80000;
+                else
```

Comment:
There is an unimplemented unit attribute for this. Better to implement that than to hardcode this.